gravitee-apim-cypress/cypress/support/index.ts

```diff
@@ -17,16 +17,18 @@
 import './common/tools.commands';
 import './common/http.commands';
 import './common/ui.commands';
+import './api-commands';
+import './application-commands';
 
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
 require('cypress-terminal-report/src/installLogsCollector')({ enableExtendedCollector: Cypress.env('printLogsToConsole') === 'always' });
 
-Cypress.Cookies.defaults({ preserve: ['Auth-Graviteeio-APIM'] });
-before(() => {
-  cy.clearCookie('Auth-Graviteeio-APIM');
-});
+// Cypress.Cookies.defaults({ preserve: ['Auth-Graviteeio-APIM'] });
+// before(() => {
+//   cy.clearCookie('Auth-Graviteeio-APIM');
+// });
 
-after(() => {
-  cy.clearCookie('Auth-Graviteeio-APIM');
-});
+// after(() => {
+//   cy.clearCookie('Auth-Graviteeio-APIM');
+// });
```

Comment:
do not merge commented things please

Reply:
Ok, good catch! It wasn't meant to be pushed, I restored the original (uncommented) version.